_includes/sidebar.html

```diff
@@ -43,6 +42,49 @@ <h3>Getting Started</h3>
         <li><a href="https://github.com/travis-ci/travis#readme">Command Line Client</a></li>
       </ul>
 
+      <h3>Customizing your Build</h3>
+      <ul>
+
+<li><a href="#Installing-a-Second-Programming-language">Installing a Second Programming language</a></li>
+<li><a href="#Build-Timeouts">Build Timeouts</a></li>
+<li><a href="#Limiting-Concurrent-Jobs">Limiting Concurrent Jobs</a></li>
+<li><a href="/user/speeding-up-the-build/">Speeding up the Build</a></li>
+
+
+<!-- group these? -->
+<li><a href="#Building-only-the-latest-commit">Building only the latest commit</a></li>
+<li><a href="#Git-Clone-Depth">Git Clone Depth</a></li>
+<li><a href="#Git-Submodules">Git Submodules</a></li>
+<li><a href="#Git-LFS">Git LFS</a></li>
+<li><a href="#Git-Sparse-Checkout">Git Sparse Checkout</a></li>
+
+<!-- group these? Build *flow* or something? -->
+<li><a href="#Building-Specific-Branches">Building Specific Branches</a></li>
+<li><a href="#Skipping-a-build">Skipping a build</a></li>
+<li><a href="#Implementing-Complex-Build-Steps">Implementing Complex Build Steps</a></li>
+
+<li><a href="#Custom-Hostnames">Custom Hostnames</a></li>
```

Comment:
these 3: custom hostnames, repo providers and what yaml version - sound like very specific questions, would it make sense to group them as or include as a reference?

Reply:
Yeah, this kinda stuff I'd like to move out of the "prime real estate" for sure.